backend/plugin/db/mssql/mssql.go

```diff
@@ -114,23 +114,32 @@ func (driver *Driver) Execute(ctx context.Context, statement string, createDatab
 		return "", io.EOF
 	}
 	batch := tsqlbatch.NewBatch(scanner)
+	execute := func(ctx context.Context, tx *sql.Tx, s string) (int64, error) {
```

Comment:
Consider moving this to a separate function. The goal is to limit the # of lines in a function, hopefully 50.